python/graphstorm/model/gnn.py

```diff
@@ -602,7 +602,8 @@ def loss_func(self):
         """
         return self._loss_fn
 
-def do_full_graph_inference(model, data, batch_size=1024, edge_mask=None, task_tracker=None):
+def do_full_graph_inference(model, data, batch_size=1024, fanout=-1, edge_mask=None,
```

Comment:
fanout is a list. its default value shouldn't be -1. probably None is a better value.

Reply:
Done.